src/NUnitFramework/framework/TestParameters.cs

```diff
@@ -44,17 +44,23 @@ public ICollection<string> Names
         /// <returns>Value of the paramter or null if not present</returns>
         public string Get(string name)
         {
-            return _parameters.ContainsKey(name) ? _parameters[name] : null;
+            return this[name];
         }
 
         public string Get(string name, string defaultValue)
         {
-            return _parameters.ContainsKey(name) ? _parameters[name] : defaultValue;
+            return this[name] ?? defaultValue;
         }
-        
-        public int Get(string name, int defaultValue)
+
+        public int GetInt(string name)
```

Comment:
I hate to be a pain, but these are all public API's and should have XMLDOC comments. Aren't they creating build warnings?